setup.py

```diff
@@ -80,7 +80,7 @@
 
         long_description=readme,
         packages=['vaxrank'],
-        package_data={'vaxrank': ['vaxrank/templates/*']},
+        package_data={'vaxrank': ['templates/*']},
```

Comment:
I'm concerned that several blog posts are so confident that this only affects `bdist` and not `sdist`.